glom/core.py

```diff
@@ -1431,6 +1448,17 @@ def __call__(self, *args, **kwargs):
             # TODO: typecheck kwarg vals?
         return _t_child(self, '(', (args, kwargs))
 
+    def __star__(self):
+        return _t_child(self, 'x', None)
+
+    def __starstar__(self):
+        return _t_child(self, 'X', None)
+
+    def __stars__(self):
+        """how many times the result will be wrapped in extra lists"""
```

Comment:
interested to see how this will be used, seems like a nice enough utility, just curious how often it comes up.

Reply:
check how `Assign` and `Delete` use it :-)  I didn't add this one because it seemed like a logical part of the basic API -- it was solving a problem

Reply:
Yeah, I should have come back and mentioned as I got further in the scroll, I saw it down there :)

Reply:
maybe I should include that in a comment "this helper is useful for unpacking nested * results without accidentally going too far"